Decompiler/Decompiler.cs

```diff
@@ -327,6 +327,14 @@ private void ProcessFile(string path, Stream stream)
                             data = ((Panorama)resource.Blocks[BlockType.DATA]).Data;
                             break;
 
+                        case ResourceType.PanoramaLayout:
+                        case ResourceType.PanoramaScript:
+                        case ResourceType.PanoramaStyle:
+                            // .vcss -> .css, .vjs -> .js, .vxml -> .xml
+                            extension = "." + extension.Substring(2, extension.Length - 2);
```

Comment:
```suggestion
                            extension = extension.Substring(1);
```

Shouldn't it be like this?